docker/otel-profgen/profgen.py

```
@@ -0,0 +1,341 @@
+
+    interval = 1.0 / RATE
+    if MAX_BATCHES > 0:
+        logger.info("Generating %d profiling event batch(es) at %.1f batches/sec", MAX_BATCHES, RATE)
+    else:
+        logger.info("Generating profiling events at %.1f batches/sec", RATE)
+
+    batches_sent = 0
+    while MAX_BATCHES <= 0 or batches_sent < MAX_BATCHES:
+        try:
+            batch = generate_event_batch()
+            success, errors = bulk(es, batch, raise_on_error=False)
```

Ruff (0.15.2)

[warning] Unpacked variable `success` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)